api/src/main/java/org/openmrs/module/queue/api/impl/QueueEntryServiceImpl.java

```diff
@@ -131,6 +131,31 @@ public QueueEntry transitionQueueEntry(QueueEntryTransition queueEntryTransition
 		return getProxiedQueueEntryService().saveQueueEntry(queueEntryToStart);
 	}
 	
+	/**
+	 * @see QueueEntryService#undoTransition(QueueEntry)
+	 */
+	@Override
+	@Transactional
```

Comment:
The @Transactional annotation here is redundant, as it is added at the class level.